setup.py

```diff
@@ -42,14 +42,10 @@
     ],
     extras_require={
         'dev': [
-            "coverage==5.5 ; python_version<'3'",
             "coverage==7.1.0 ; python_version>'3'",
```

Comment:
Do we need to retain the `python_version>'3'` condition if we've dropped support for 2.x?

Reply:
nope, I've removed these all.